### PR TITLE
chore: update dev mainnet url

### PR DESCRIPTION
**Pull request by mchappell**
Update Lace to use `dev-mainnet` instead of `dev-perf-mainnet`

### CODE REVIEW

**Review by mkazlauskas — APPROVED** [review]

File: apps/browser-extension-wallet/.env.defaults [hunk 1]
```diff
@@ -65,7 +65,7 @@ POSTHOG_DEV_TOKEN_PREPROD=phc_Xlmldm6EYSfQVgB9Uxm3b2xC1noDlgFFXpF9AJ6SMfJ
 POSTHOG_DEV_TOKEN_PREVIEW=phc_e8SaOOWpXpNE59TnpLumeUjWm4iv024AWjhQqU406jr
 
 # Cardano Services
-CARDANO_SERVICES_URL_MAINNET=https://backend.dev-perf-mainnet.eks.lw.iog.io
+CARDANO_SERVICES_URL_MAINNET=https://backend.dev-mainnet.eks.lw.iog.io
```
Comment (gytis-ivaskevicius) [inline apps/browser-extension-wallet/.env.defaults:68]:
```suggestion
CARDANO_SERVICES_URL_MAINNET=https://live-mainnet.lw.iog.io
```

File: apps/browser-extension-wallet/.env.developerpreview [hunk 2]
```diff
@@ -59,7 +59,7 @@ POSTHOG_DEV_TOKEN_PREPROD=phc_Xlmldm6EYSfQVgB9Uxm3b2xC1noDlgFFXpF9AJ6SMfJ
 POSTHOG_DEV_TOKEN_PREVIEW=phc_e8SaOOWpXpNE59TnpLumeUjWm4iv024AWjhQqU406jr
 
 # Cardano Services
-CARDANO_SERVICES_URL_MAINNET=https://backend.dev-perf-mainnet.eks.lw.iog.io
+CARDANO_SERVICES_URL_MAINNET=https://backend.dev-mainnet.eks.lw.iog.io
```
Comment (gytis-ivaskevicius) [inline apps/browser-extension-wallet/.env.developerpreview:62]:
```suggestion
CARDANO_SERVICES_URL_MAINNET=https://live-mainnet.lw.iog.io
```

File: apps/browser-extension-wallet/.env.defaults (hunk 1 above)
Comment (mchappell) [reply apps/browser-extension-wallet/.env.defaults:68]: we need to be using `dev` until we've done the next live deployment to make use of later cardano-services package

File: apps/browser-extension-wallet/.env.developerpreview (hunk 2 above)
Comment (mchappell) [reply apps/browser-extension-wallet/.env.developerpreview:62]: we need to be using `dev` until we've done the next live deployment to make use of later cardano-services package

File: apps/browser-extension-wallet/.env.defaults (hunk 1 above)
Comment (gytis-ivaskevicius) [reply apps/browser-extension-wallet/.env.defaults:68]: Sorry, typo

File: apps/browser-extension-wallet/.env.developerpreview (hunk 2 above)
Comment (gytis-ivaskevicius) [reply apps/browser-extension-wallet/.env.developerpreview:62]: Sorry, typo

**Review by gytis-ivaskevicius — APPROVED** [review]

**Pull request merged**